stream-chat-android-sample/src/main/kotlin/io/getstream/chat/sample/feature/channels/ChannelsFragment.kt

```diff
@@ -15,6 +15,7 @@ import com.getstream.sdk.chat.viewmodel.channels.bindView
 import com.getstream.sdk.chat.viewmodel.factory.ChannelsViewModelFactory
 import io.getstream.chat.sample.R
 import io.getstream.chat.sample.common.navigateSafely
+import kotlinx.android.synthetic.main.fragment_channel.*
```

Comment:
This looks really suspicious, with no other changes in the file.

But also: [pls no](https://github.com/GetStream/stream-chat-android/pull/756) 😅  (if it's actually needed, maybe rebase after that PR is merged)

Reply:
Ups, some conflict reloving left-over 🤦🏼